client/src/components/AppFrame/AppFrame.js

```diff
@@ -92,7 +93,9 @@ const AppFrame = (props) => ({
       )
     } else if(currentView === 'imageScrubber') {
       tabContents = (
-        <ImageScrubber />
+        <Infinite containerHeight={200} elementHeight={40}>
```

Comment:
I think it's probably worth following how the file does it, by adding styles into the styles object

Reply:
Adding these rules to the styles object breaks the module, as the `containerHeight` and `elementHeight` module properties are required and break. Since the styles aren't available when the `react-infinite` module is loaded into the view.

I've wrapped the styles in a separate object to make available and follow the same pattern

Reply:
yep as soon as I suggested this, I also thought it might not work at all hehe (been doing more RN development recently, but in that world you just pass a fat `styles` object) instead of props like this